### DISCUSSION TITLE
Is there a Release cycle?

### DISCUSSION THREAD

**Discussion by tqwewe** [open-telemetry/opentelemetry-rust#1031] [General]
I'm wondering if there's any kind of release cycle for Opentelemetry-Rust?

I noticed that only minor versions are mostly published, but wonder if there might be any plans for releasing patch versions more frequently?

The main reason I'm asking is that there are some new changes in the main branch which I depend on in another project waiting to be published.

Thanks!

**Comment by TommyCpp** [comment]
One thing I will call out is since we are pre 1.0 minor version is still breaking. 

I personally support a release cycle for non breaking change(in current state, it means a patch version). But I see two reasons that may block this.
- The contribution/changes we receive varies so if we follow a time based release cycle we may not releasing much
- I am still concerned with our integration test coverage that may prevent us to release non breaking changes as it will break ppl's build instantly if we are not careful(compare with minor version bump, which requires manual upgrade)

Overall I think release cycle or more frequent release is the right thing to do but may need some work to make it happen

**Comment by djc** [comment]
I think we've put out patch versions in response to requests before, so just file an issue asking about that? I agree with @TommyCpp that it might not currently make to sense to put out releases on a regular cadence.

**Comment by cijothomas** [comment]
Closing this, as we have announced dates for future releases : https://github.com/open-telemetry/opentelemetry-rust/issues/1678
